psi4/driver/p4util/numpy_helper.py

```diff
@@ -482,6 +482,11 @@ def _chain_dot(*args, **kwargs):
 
     return ret
 
+def _irrep_access(self, *args, **kwargs):
```

Comment:
I like this. It should help with debugging in the future.

Reply:
Agreed, this is cool